File: php/render_template.php
<?php
require_once('vendor/autoload.php');

use Docxmerge\Docxmerge;

$docxmerge = new Docxmerge("API_KEY");

$fp = fopen("./hello_world_template_php.pdf", "w");

$docxmerge->renderTemplate(
    $fp,
    "hello-world",
    array(
        "name" => "James bond",
        "logo" => "https://docxmerge.com/assets/android-chrome-512x512.png"
    ),
    "PDF",
    "latest"
);

File: python/render_template.py
# pip install docxmerge_sdk 
# https://pypi.org/project/docxmerge-sdk/

from docxmerge_sdk import Docxmerge
docxmerge = Docxmerge("API_KEY", "default")

pdf = docxmerge.render_template(
    "hello-world",
    {
        "logo": "https://docxmerge.com/assets/android-chrome-512x512.png",
        "name": "James Bond"
    },
    "PDF",
    "latest"
)

open("./example-hello-world-template.pdf", "wb").write(pdf)
